ingestion/src/metadata/profiler/metrics/static/min_length.py

```diff
@@ -67,7 +67,7 @@ def df_fn(self, dfs=None):
             min_length_list = []
 
             for df in dfs:
-                if any(df[self.col.name]):
+                if any(df[self.col.name].dropna()):
```

Comment:
same comment here if we drop `na` we need to state it clearly

Reply:
will do the same 👍